standalone/src/models/cart/cart-discount/cart-discount-target/fields-config.ts

```diff
@@ -0,0 +1,46 @@
+import { fake, type TModelFieldsConfig } from '@/core';
+import { targetTypes } from './constants';
+import type {
+  TCartDiscountTargetGraphql,
+  TCartDiscountTargetRest,
+} from './types';
+
+// https://docs.commercetools.com/api/projects/cartDiscounts#cartdiscounttarget
+
+const targetTypeToGraphqlTypenameMap: Record<string, string> = {
+  customLineItems: 'CustomLineItemsTarget',
+  lineItems: 'LineItemsTarget',
+  pattern: 'PatternTarget',
+  shipping: 'ShippingTarget',
+  totalPrice: 'TotalPriceTarget',
+};
+
+const commonFieldsConfig = {
+  type: fake((f) => f.helpers.arrayElement(Object.values(targetTypes))),
+};
+
+export const restFieldsConfig: TModelFieldsConfig<TCartDiscountTargetRest> = {
+  fields: {
+    ...commonFieldsConfig,
+  },
+};
+
+export const graphqlFieldsConfig: TModelFieldsConfig<TCartDiscountTargetGraphql> =
+  {
+    fields: {
+      ...commonFieldsConfig,
+      predicate: null,
+      __typename: null,
+    },
+    postBuild: (model) => {
```

Comment:
I tried to replicate what we already have in the monorepo ([reference](https://github.com/commercetools/merchant-center-frontend/blob/main/packages-shared/test-data/src/cart-discount-target/transformers.js)).